QFXparser/QFXparser.cs

```diff
@@ -123,6 +148,20 @@ private RawStatement BuildRaw()
             return _statement;
         }
 
+        private object ConvertQfxType(string content, Type targetType)
+        {
+            object result;
+            if (targetType == typeof(DateTime))
+            {
+                result = ParsingHelper.ParseDate(content);
+            }
+            else
+            {
+                result = Convert.ChangeType(content, targetType);
```

Comment:
Looks like `Convert.ChangeType` can throw an exception in case of bad formatted content. I would suggest to wrap this section in a `try..catch` and return the default type value in case of exception.

Reply:
wrapped it as per your suggestion and added an invalid Amount value to the test, for catching this scenario.